packages/types/src/signIn.ts

```diff
@@ -83,6 +84,8 @@ export interface SignInResource extends ClerkResource {
   authenticateWithMetamask: () => Promise<SignInResource>;
 
   createMagicLinkFlow: () => CreateMagicLinkFlowReturn<SignInStartMagicLinkFlowParams, SignInResource>;
+
+  startSamlFlow: (params: StartSamlFlowParams) => Promise<void>;
```

Comment:
🔧 If we don't merge it with `authenticateWithRedirect` it should be renamed to `authenticateWithSaml` to follow the authenticateWithX naming convention.

Reply:
Merged with `authenticateWithRedirect`.